docs/CONFIGURATION.md

```diff
@@ -16,7 +16,7 @@ The following arguments are required to run `ld-find-code-refs`. For a detailed
 | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `accessToken` | LaunchDarkly [personal access token](https://docs.launchdarkly.com/docs/api-access-tokens) with writer-level access, or access to the `code-reference-repository` [custom role](https://docs.launchdarkly.com/v2.0/docs/custom-roles) resource |
 | `dir`         | Path to existing checkout of the git repo. The currently checked out branch will be scanned for code references.                                                                                                                               |
-| `projKey`     | A LaunchDarkly project key.                                                                                                                                                                                                                    |
+| `projKey`     | A LaunchDarkly project key. Cannot be used with `projects` block.                                                                                                                                                                                                                   |
```

Comment:
The intent here is to make clear that either projKey or projects is required, so maybe we can improve this readme like so:
```
| `projKey` or `projects` | A LaunchDarkly project key or a projects block containing a list of project keys. See Advanced configuration for more details.                                                                                               |
```
What do you think?